independent-publisher-connectors/snowflake/apiDefinition.swagger.json

```diff
@@ -0,0 +1,745 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Snowflake REST",
+    "description": "Snowflake connector based on Snowflake REST service",
+    "version": "1.0",
+	"contact": {
+      "name": "Rene Koch",
+      "url": "https://github.com/rekodus",
+      "email": "rene.koch@uniper.energy"
+    }
+  },
+  "host": "test.azure.snowflakecomputing.com",

```

Comment:
@rekodus Please provide a valid host URL, if you are using a policy template for host URL, please provide it like `"yourinstance.snowflakecomputing.com"`

Reply:
what is the point of this if it is replaced anyhow?  I am hesistant to put my compay's instance name here. is it need for automated testing? or would it be sufficient to put "snowflakecomputing.com"?